packages/block-library/src/spacer/edit.js

```diff
@@ -48,7 +49,7 @@ const SpacerEdit = ( {
 
 	return (
 		<>
-			<div { ...useBlockProps() }>
```

Comment:
Ideally we should avoid this and make the wrapper itself resizable. I'll look into that.

Reply:
yes, I tried it and I believe there's something with the "ref" not being forwarded properly to the third-party dependency.